app/gui/index.html

```diff
@@ -38,6 +38,15 @@
             user-scalable = no"
     />
     <title>Enso %ENSO_IDE_VERSION%</title>
+
+    <script type="module">
+      import { platform } from  '../common/src/detect.ts'
+
+      const currentPlatform = platform()
+
+      document.documentElement.setAttribute('data-version', '%ENSO_IDE_VERSION%')
+      document.documentElement.classList.add(currentPlatform.toLowerCase())
```

Comment:
scuff

can we prefix the platform name please? so like `platform-macos`

Reply:
Why?

Reply:
i guess mostly to indicate that they are variants of a single property (meaning that there shouldn't be more than one of `platform-` on any element)

Reply:
Not sure what you mean. I can specify 'macos:text-12 windows:text-24 linux:text-10`. Yes, they exclude each other, but i still don't understand how's `platform` relates to it

Reply:
oh true :skull: this is a modifier/variant, not a property

but i think regardless it still makes sense to prefix the classes/variants just to make it clear that they are related to each other. just my two cents

also any prefix works, it doesn't have to be platform